nuxt/server/api/auth-proxy.ts

```diff
@@ -0,0 +1,60 @@
+import { H3Event, readBody } from 'h3'
+import consola from 'consola'
+import { TURNSTILE_HEADER_KEY } from '~/utils/constants'
+
+export default defineEventHandler(async function (event: H3Event) {
+  const { req, res } = event.node
+  consola.debug(getMethod(event))
+  consola.debug('Request: ' + req)
+  let body: any
+  try {
+    body = await readBody(event)
+  } catch (e) {
+    consola.error(e)
+  }
+
+  switch (body.operationName) {
```

Comment:
Should not be limited, I'd say.

Reply:
If it wasn't limited then all requests that go to the database would need a turnstile verification which means that every page would need a turnstile element to perform the captcha challenge. We should figure out where we want a captcha-like check and where we don't need it

Reply:
I agree :+1: